src/main/java/klaxon/klaxon/novisoculis/CubeModel.java

```diff
@@ -0,0 +1,207 @@
+package klaxon.klaxon.novisoculis;
```

Comment:
Let's keep these under the angelica namespace; might also consider anything that needs to be usable externally to have it available under the API now that we're publishing an API package as well.